CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## v1.3.2 - 2025-02-27
```

Comment:
I think we should just skip 1.3.1, so change the previous changelog entry to 1.3.2 rather than an empty version